realsense2_camera/cfg/rs415_params.cfg

```diff
@@ -12,8 +12,8 @@ base_d400_params.add_base_params(gen, "rs415_")
 
 #             Name                               Type       Level  Description                  Default    Min     Max
 gen.add("rs415_depth_enable_auto_white_balance", bool_t,    9,     "Enable Auto White Balance", False)
-gen.add("rs415_depth_exposure",                  int_t,     10,    "Exposure",                  1650,      1,      8300) # The exposure step is 20 by definition but the param step is 1 by default so we divided all values by 20 and multiply it back in SW
-gen.add("rs415_depth_laser_power",               double_t,  11,    "Laser Power",               12.5,      0,      12) # Multiple value by 30
+gen.add("rs415_depth_exposure",                  int_t,     10,    "Exposure",                  3000,      1,      8300) # The exposure step is 20 by definition but the param step is 1 by default so we divided all values by 20 and multiply it back in SW
+gen.add("rs415_depth_laser_power",               double_t,  11,    "Laser Power",               4.0,      0,      12.5) # Multiple value by 30
```

Comment:
Why are you needing to modify the max? It seems like we wouldn't want to change the min and max.

Reply:
For some reason the max was 12 but the old default was 12.5, I just wanted to make it so we could go back to the old default if we wanted to

Reply:
I don't think that's how this works. I think it would be capped at 12. Let's change this back.

Reply:
I did validate that it was getting set outside the limit by printing out the value in the code